extra $TESTCASE_ROOT print since #7703

Since #7703 got merged, I'm seeing local consistent failures like this. This is very strange since it looks fairly unrelated. Do you have an idea what this could be?

```diff
--- a/_build/.sandbox/ab08f82e1b66bb77b75f14bbbc41fe2c/default/test/blackbox-tests/test-cases/action-runner/build-e2e.t
+++ b/_build/.sandbox/ab08f82e1b66bb77b75f14bbbc41fe2c/default/test/blackbox-tests/test-cases/action-runner/build-e2e.t.corrected
@@ -18,4 +18,5 @@ dune command
   # mkdir -p _build/default/ar1;cd _build/default/ar1;echo -n xxx > aaa
   $ ar2="_build/ar2.*.log"
   $ grep bbb $ar2
+  # $TESTCASE_ROOT
   # mkdir -p _build/default/ar2;cd _build/default/ar2;echo -n yyy > bbb
make: *** [Makefile:102: test] Error 1   
```

_Originally posted by @emillon in https://github.com/ocaml/dune/issues/7703#issuecomment-1557323433_

--- a/test/blackbox-tests/test-cases/action-runner/build-e2e.t
+++ b/test/blackbox-tests/test-cases/action-runner/build-e2e.t
@@ -14,8 +14,8 @@ dune command
 
   $ timeout 2 dune internal action-runner build --runner ar1 --runner ar2
   $ ar1="_build/ar1.*.log"
-  $ grep aaa $ar1
+  $ grep -e mkdir -e aaa $ar1
   # mkdir -p _build/default/ar1;cd _build/default/ar1;echo -n xxx > aaa
   $ ar2="_build/ar2.*.log"
-  $ grep bbb $ar2
+  $ grep -e mkdir -e bbb $ar2
   # mkdir -p _build/default/ar2;cd _build/default/ar2;echo -n yyy > bbb

